roles/influxdb/tasks/bucket.yml

```
@@ -0,0 +1,46 @@
+---
+- name: Install Influx DB 2 CLI
+  package: 
+    name: influxdb2-cli
+
+- name: Check bucket
+  command: influx bucket list -n "{{ influxdb_bucket }}"
+  register: result
+  ignore_errors: yes
+
+- name: Influx DB 2 initial setup
+  command: influx setup \
+    --org "{{ influxdb_org }}" \
+    --bucket "{{ influxdb_bucket }}" \
+    --username "{{ influxdb_user }}" \
+    --password "{{ influxdb_password }}" \ 
+    --token "{{ influxdb_token }}" \
+    --host http://{{ influxdb_host }}:8086 \
+    --force 
```

```suggestion
    --force
```
Remove the trailing space in the line. Linters will always find them. :-)